### PR TITLE
update word_embeddings tutorial

### PR DESCRIPTION
**Pull request by omarsar**
I think the comment: `# Each tuple is ([ word_i-CONTEXT_SIZE, ..., word_i-1 ], target word)` might be misleading. The current code outputs tuples of the form: `([word_i-1,...,word_i-CONTENT_SIZE], target word)`.

If we wanted tuples of the form `([ word_i-CONTEXT_SIZE, ..., word_i-1 ], target word)` then we just need to reverse the range. I also noticed that some previous code was producing the correct form. Can someone please double-check this?

### CODE REVIEW

File: beginner_source/nlp/word_embeddings_tutorial.py [hunk 1]
```diff
@@ -211,7 +211,7 @@
 # Each tuple is ([ word_i-CONTEXT_SIZE, ..., word_i-1 ], target word)
 ngrams = [
     (
-        [test_sentence[i - j - 1] for j in range(CONTEXT_SIZE)],
+        [test_sentence[i - j - 1] for j in reversed(range(CONTEXT_SIZE))],
```
[the remark this answers is not in the document]
Comment (omarsar) [reply beginner_source/nlp/word_embeddings_tutorial.py:214]: Another question: If we have the sentence "When forty winters shall besiege thy brow..." how do we want our ngrams tuples?

A: `(['forty', 'When'], 'winters'), (['winters', 'forty'], 'shall'), (['shall', 'winters'], 'besiege')...`

OR

B: `(['When', 'forty'], 'winters'), (['forty', 'winters'], 'shall'), (['winters', 'shall'], 'besiege')...`

from the comment in the tutorial (`# Each tuple is ([ word_i-CONTEXT_SIZE, ..., word_i-1 ], target word)`) I expect we would like B. Or maybe I am totally confusing this.

[the remark this answers is not in the document]
Comment (omarsar) [reply beginner_source/nlp/word_embeddings_tutorial.py:214]: @albanD could you look into my question above?

[the remark this answers is not in the document]
Comment (albanD) [reply beginner_source/nlp/word_embeddings_tutorial.py:214]: Hey!
Yes that makes sense. Note that the second example has the same issue where words from before are flipped in the same way.
I don't think it makes any difference for training though. Did you check locally that running the code still show decreasing loss?

[the remark this answers is not in the document]
Comment (svekars) [reply beginner_source/nlp/word_embeddings_tutorial.py:214]: What's the final on this?

[the remark this answers is not in the document]
Comment (omarsar) [reply beginner_source/nlp/word_embeddings_tutorial.py:214]: we can close this. Thanks @svekars

**Pull request closed without merging**